web/scadnano-styles.css

```diff
@@ -735,7 +735,7 @@ https://stackoverflow.com/questions/47758565/adding-fedropshadow-to-a-vertical-l
     visibility: visible;
 }
 
-.potential-crossover {
+.potential-segment {
```

Comment:
Add a comment that the CSS selector potential-segment captures both potential crossovers and potential extensions.